### PR TITLE
Created API end point for Members and Family Members

### PR DESCRIPTION
**Pull request by sourabh-sawant-v2**
Created: API end point for Members and Family Members controller.
Added: added Rspec unit test cases for both controller, with an request format for both API call for JSON or HTML template with 
             web request.
Added: regex expression for both member and family_member model, for their attributes.

### CODE REVIEW

File: app/controllers/login_controller.rb [hunk 1]
```diff
@@ -5,28 +5,26 @@ def new
...
-            token = SecureRandom.hex()
-            session = Session.create(user_id: user.id, token:token)
-            cookies.signed[:session_token] = { value: token, httponly: true, secure: Rails.env.production? }
-            redirect_to pages_path, notice: "Login Successfully"
-            # render json: {token:token, user_id: session[:user_id]}
-        else 
-            redirect_to root_path, notice: "Invalid username or password"
-            # render :new, notice: "login failed"
-        end 
+                session[:user_id] = user.id
```
Comment (shilesh) [inline app/controllers/login_controller.rb]: wrong indentation  It should be only 2 spaces

File: spec/controllers/api_response/family_members_spec.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            expect(response.content_type).to eq("application/json; charset=utf-8")
+          end
+    end
+
+
+    it "update member in application/json format successfully" do
+        fm = FamilyMember.create!(name: 'Test5', mobile: '5694549248', blood_group: 'A-')
+        patch :update, params: { id: fm.id, family_member: { name: 'sst' } }, :format => :json
+       
+        expect(response.status).to be(200)
```
Comment (shilesh) [inline spec/controllers/api_response/family_members_spec.rb:49]: Check the data along with status, like name, email ... etc

File: spec/controllers/api_response/login_spec.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe LoginController, type: :request do
+
+    context "login successfully" do
+    before {Admin.create!(name:'test', mobile:'123', password:'123')}
+        it {
```
Comment (shilesh) [inline spec/controllers/api_response/login_spec.rb]: use `do ... end` instead of `{... }`

**Review by shilesh — CHANGES_REQUESTED** [review]
Correct indentation and update test cases, added comments.

File: app/controllers/login_controller.rb (hunk 1 above)
Comment (sourabh-sawant-v2) [reply app/controllers/login_controller.rb]: indentation is corrected

File: spec/controllers/api_response/family_members_spec.rb (hunk 2 above)
Comment (sourabh-sawant-v2) [reply spec/controllers/api_response/family_members_spec.rb:49]: updated the test cases, with all 'get' requests some property of data is validated to be present in response

File: spec/controllers/api_response/login_spec.rb (hunk 3 above)
Comment (sourabh-sawant-v2) [reply spec/controllers/api_response/login_spec.rb]: removed { . . } block, and replaced with do-end block

**Pull request merged**